exalearn-ui/src/app/interfaces/question.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface Question {
+	id: number;
+	level: string;
+	audioId?: number;
+	audioFile?: string;
+	type: string;
+	text: string;
+	description?: string;
+	score: number;
+	answers?: string[];
+}
```

Comment:
Discussed on the standup.
Please change.

Reply:
Got it, I'll change it